dotnet/sample-apps/aws-sdk/wrapper/SampleApps/AwsSdkSample.sln

```diff
@@ -1,25 +0,0 @@
-﻿
-Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
While I don't know why this file is checked in... I don't think this is the right PR to remove it.

Reply:
Ok, I'll close this PR then to have a fresh start with new clone then

Reply:
That shouldn't be necessary.  Just undo deleting this file.